posthog/models/feedback/survey.py

```diff
@@ -66,6 +67,13 @@ class Meta:
     archived: models.BooleanField = models.BooleanField(default=False)
     # It's not a strict limit as it's enforced in a periodic task
     responses_limit = models.PositiveIntegerField(null=True)
+    events = ArrayField(
```

Comment:
hmmmmm, given my comment on posthog-js about the events config here needing to be fleshed out further, and not making any breaking changes, I'm tempted to put this field in `conditions` - I don't think we'd need to query this alone or do anything with it other than in posthog-js. It also fits well in there alongside other conditions for filtering surveys.

Reply:
sounds good to me, I can move the `events` collection into the `conditions`  JSON struct